### ISSUE TITLE
Won't work on startup

### ISSUE THREAD

**Issue by NZiozis** [itchyny/lightline.vim#298]
Hi. I'm a little new to vim and all the plugins and I'm using Vundle to install this one. When I open gvim, powerline doesn't show up, but when I do a :PluginInstall or Update then it appears. Is there a way to skip this step so that the line appears on startup?
set nocompatible
filetype off

"set the runtime path to include vundle and initialize
set rtp+=/usr/share/vim/bundle/Vundle.vim

call vundle#begin('/usr/share/vim/addons/plugin')

"Enhanced sytax highlighting for python
Plugin 'hdima/python-syntax'

"Let vundle handle itself
Plugin 'VundleVim/Vundle.vim'

"The rest of the plugins
Plugin 'itchyny/lightline.vim'
Plugin 'NLKNguyen/papercolor-theme'
Plugin 'terryma/vim-multiple-cursors' 
Plugin 'tpope/vim-surround' 
"Figure this one out for a tree heirarchy on the side
"Plugin 'scrooloose/nerdtree'

"Figure out how to install this one for fuzzy finder
"Plugin 'junegunn/fzf.vim'
"Plugin '/home/nikoziozis/Desktop/PersonalProjects/dotfiles/dotbot/bin/fzf' 

call vundle#end()
filetype plugin indent on

" Make external commands work through a pipe instead of a pseudo-tty
"set noguipty

"This is the settings for python-syntax highlighting
let python_highlight_all = 1

" You can also specify a different font, overriding the default font
set guifont=Ubuntu\ Mono\ 18 

"set the background color and theme here
"This line is here in case the theme messes up. Just uncommnet this and it
"should work.       g:PaperColor_Theme_Options
set background=dark
colorscheme PaperColor

"This turns on the line numbers on the side
set number 
set laststatus=2
set tabstop=2

"This modifies the gui 
set guioptions-=T 
set guioptions-=m
set guioptions-=r

"begining of custom keymaps

"maps going forward and backwards a tab to <F7> and <F8>
map <F7> :tabp<Enter>   
map <F8> :tabn<Enter>
nnoremap <silent> <F5> :!python3 %<CR>

"maps the changing the active split screen from combo commands to simpler ones
nnoremap <C-J> <C-W><C-J>
nnoremap <C-K> <C-W><C-K> 
nnoremap <C-L> <C-W><C-W>
nnoremap <C-H> <C-W><C-H> 

"end of custom key maps

" Source a global configuration file if available
if filereadable("/etc/vim/gvimrc.local")
  source /etc/vim/gvimrc.local
endif

**Comment by itchyny** [comment]
Please paste your entire vimrc, or minimal vimrc to reproduce the problem if possible so that I can investigate.

**Comment by NZiozis** [comment]
set nocompatible
filetype off

"set the runtime path to include vundle and initialize
set rtp+=/usr/share/vim/bundle/Vundle.vim

call vundle#begin('/usr/share/vim/addons/plugin')

"Enhanced sytax highlighting for python
Plugin 'hdima/python-syntax'

"Let vundle handle itself
Plugin 'VundleVim/Vundle.vim'

"The rest of the plugins
Plugin 'itchyny/lightline.vim'
Plugin 'NLKNguyen/papercolor-theme'
Plugin 'terryma/vim-multiple-cursors' 
Plugin 'tpope/vim-surround' 
"Figure this one out for a tree heirarchy on the side
"Plugin 'scrooloose/nerdtree'

"Figure out how to install this one for fuzzy finder
"Plugin 'junegunn/fzf.vim'
"Plugin '/home/nikoziozis/Desktop/PersonalProjects/dotfiles/dotbot/bin/fzf' 

call vundle#end()
filetype plugin indent on

" Make external commands work through a pipe instead of a pseudo-tty
"set noguipty

"This is the settings for python-syntax highlighting
let python_highlight_all = 1

" You can also specify a different font, overriding the default font
set guifont=Ubuntu\ Mono\ 18 

"set the background color and theme here
"This line is here in case the theme messes up. Just uncommnet this and it
"should work.       g:PaperColor_Theme_Options
set background=dark
colorscheme PaperColor

"This turns on the line numbers on the side
set number 
set laststatus=2
set tabstop=2

"This modifies the gui 
set guioptions-=T 
set guioptions-=m
set guioptions-=r

"begining of custom keymaps

"maps going forward and backwards a tab to <F7> and <F8>
map <F7> :tabp<Enter>   
map <F8> :tabn<Enter>
nnoremap <silent> <F5> :!python3 %<CR>

"maps the changing the active split screen from combo commands to simpler ones
nnoremap <C-J> <C-W><C-J>
nnoremap <C-K> <C-W><C-K> 
nnoremap <C-L> <C-W><C-W>
nnoremap <C-H> <C-W><C-H> 

"end of custom key maps

" Source a global configuration file if available
if filereadable("/etc/vim/gvimrc.local")
  source /etc/vim/gvimrc.local
endif

**Comment by itchyny** [comment]
It works well for me on Ubuntu with you configuration but

- The `maps going forward and backwards a tab to and` section (following three lines) is unnecessary or invalid configuration.
- The `maps the changing the active split screen from combo...` section is invalid.
- I'm not sure what's written in `/etc/vim/gvimrc.local` so appreciate if you look into the file too.

**Comment by NZiozis** [comment]
Ok. Thank you. As I mentioned I am new to vim and this might be something completely on my end that is causing the issue. The only reason why I noticed that this plugin wasn't working is because of the really large graphical component. I appreciate the time that you took looking into the issue and pointing out mistakes that I made in my configuration. I'm going to mark this as closed because this is definitely a problem with how I set up vim and not the plugin itself.
